src/test/java/net/oauth/jsontoken/JsonTokenTest.java

```diff
@@ -1,28 +1,126 @@
 package net.oauth.jsontoken;
 
+import com.google.gson.JsonObject;
 import net.oauth.jsontoken.crypto.HmacSHA256Signer;
+import net.oauth.jsontoken.crypto.SignatureAlgorithm;
+import org.joda.time.Duration;
+import org.joda.time.Instant;
+
+import java.security.SignatureException;
 
 public class JsonTokenTest extends JsonTokenTestBase {
 
-  public void testCreateJsonToken() throws Exception {
+  private static final String TOKEN_STRING_NULL_FIELDS = "eyJhbGciOiJIUzI1NiJ9.eyJoZWxsbyI6IndvcmxkIn0.URnYdSXdAAEukebqZgMq6oFjK4E9cEZlfvO8tBe_WeA";
+  private static final String TOKEN_STRING_EMPTY_PAYLOAD = "eyJhbGciOiJIUzI1NiJ9.e30.XmNK3GpH3Ys_7wsYBfq4C3M6goz71I7dTgUkuIa5lyQ";
+
+  public void testSignAndSerialize() throws Exception {
     HmacSHA256Signer signer = new HmacSHA256Signer("google.com", "key2", SYMMETRIC_KEY);
 
     JsonToken token = new JsonToken(signer, clock);
     token.setParam("bar", 15);
     token.setParam("foo", "some value");
     token.setAudience("http://www.google.com");
     token.setIssuedAt(clock.now());
-    token.setExpiration(clock.now().withDurationAdded(60, 1));
+    token.setExpiration(clock.now().plus(Duration.standardSeconds(1)));
 
     assertEquals(TOKEN_STRING, token.serializeAndSign());
   }
 
-  public void testCreateAnotherJsonToken() throws Exception {
+  public void testSignAndSerializeNullFields() throws Exception {
```

Comment:
We use underscores to separate logical components of the test method name: https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names

e.g. testSignAndSerialize_nullFields, testSignAndSerialize_emptyPayload.

Reply:
Renamed